src/main/java/com/example/QuizGame/controller/QuestionController.java

```diff
@@ -101,6 +98,9 @@ public String answerQuestion() {
     public String congratulations(Model model) {
         String username = (String) httpSession.getAttribute("username");
         model.addAttribute("username", username);
```

Comment:
If spring security provides required sanitization, please ignore this message.
You should sanitize user inputs especially when it using save/update functions.